command/ca/provisioner/add.go

```diff
@@ -252,6 +268,11 @@ $ step ca provisioner add acme-smallstep --type ACME --ca-config ca.json
 Add an X5C provisioner.
 '''
 $ step ca provisioner add x5c-smallstep --type X5C --x5c-root x5cRoot.crt
+'''
+
+Add a K8s Service Account provisioner.
+'''
+$ step ca provisioner add my-kube-provisioner --type K8sSA --pem-keys keys.pub
```

Comment:
Can we add to the docs how to get the keys?

Reply:
I have no idea how to get these keys. I don't think they're accessible w/out using the TokenReview API. I think mike's intention of offering this as v1 was to just have "something" out there that we could iterate on quickly for a v2.

Reply:
sounds good.